src/modules/auth/auth.service.ts

```diff
@@ -1,28 +1,30 @@
-import {Injectable, UnauthorizedException} from '@nestjs/common';
-import {JwtService} from '@nestjs/jwt';
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
 
-import {ConfigService} from '../config/config.service';
-import {User} from '../user/user.entity';
-import {UsersService} from '../user/user.service';
-import {LoginPayload} from './login.payload';
+import { ConfigService } from '../config/config.service';
+import { User } from '../user/user.entity';
+import { UsersService } from '../user/user.service';
+import { LoginPayload } from './login.payload';
 
 @Injectable()
 export class AuthService {
   constructor(
     private readonly jwtService: JwtService,
     private readonly configService: ConfigService,
     private readonly userService: UsersService,
-  ) {
-  }
+  ) {}
 
   async generateToken(user: User) {
+    const userPayload = user;
+    delete userPayload.password;
```

Comment:
password should not be in plain text in the token (only plain on register then, not on login)